dashboard/config/locales/en.yml

```diff
@@ -177,8 +177,6 @@ en:
         subject: "Join a class with %{teacher_name} at Code.org!"
       student_disassociated:
         subject: "Code.org notification: %{student_name} is no longer associated as your student."
-      teacher_disassociated:
```

Comment:
Not having really touched i18n before... Is this the appropriate file to remove the string from? It'll then be auto-removed from other files? Do I need to take action to remove it from other files?